scripts/start.py

```diff
@@ -142,6 +143,8 @@ def main(args=None):
         # Give webpack few seconds to do the initial compilation.
         time.sleep(10)
 
+        common.start_redis_server()
```

Comment:
optional: should we run this in the background? [I'm slightly leaning toward moving this to the background but if it makes complicated to use this function in other files than you can leave it as-is.)